include/ITS.Propagation.LFMF/LFMF.h

```diff
@@ -0,0 +1,109 @@
+/** @file LFMF.h
+ * Interface header for this library
+ */
+#ifndef __ITS_PROPAGATION_LFMF_LFMF_H__
+#define __ITS_PROPAGATION_LFMF_LFMF_H__
+
+#include <complex>
+#include <cfloat>
+
+#include "LFMFConfig.h"
+#include "ReturnCodes.h"
+
+using std::complex;
+using std::abs;
+
+namespace ITS {
+namespace Propagation {
+namespace LFMF {
+
+// Define cross-platform DLLEXPORT
+// Export the DLL functions as "C" and not C++
+#ifndef DOXYGEN_SHOULD_SKIP
+    #ifdef _WIN32
+        #define DLLEXPORT extern "C" __declspec(dllexport)
+    #else
+        #define DLLEXPORT extern "C"
+    #endif
+#endif
+
+#define MAX(x, y) (((x) > (y)) ? (x) : (y))                 ///< A macro that returns the maximum of \a a and \a b.
+#define MIN(x, y) (((x) < (y)) ? (x) : (y))                 ///< A macro that returns the minimum of \a a and \a b.
+#define DIM(x, y) (((x) > (y)) ? (x - y) : (0))             ///< A macro that returns the difference of \a a and \a b.
+
+#define PI                          3.1415926535897932384   ///< Approximate value of @f$ \pi @f$
```

Comment:
Please replace the macros from lines 31-44 with properly-typed `constexpr` constants. For example:

```cpp
constexpr double PI = 3.1415;
```